src/ert/gui/ertwidgets/caselist.py

```diff
@@ -20,23 +21,31 @@
 
 class AddRemoveWidget(QWidget):
     """
-    A simple class that provides to vertically positioned buttons for adding and
-    removing something.  The addFunction and removeFunction functions must be
-    provided.
+    A widget providing add and remove buttons.
+
+    Parameters
+    ----------
+    addFunction: Callable to be connected to the add button.
+    removeFunction: Optional Callable to be connected to the remove button.
+        If None, the remove button is not created.
     """
 
-    def __init__(self, addFunction=None, removeFunction=None):
-        QWidget.__init__(self)
+    def __init__(
+        self, addFunction: Callable, removeFunction: Optional[Callable] = None
```

Comment:
Suggest just removing this argument instead. We would like to add a remove function, but not sure this would be the way. In any case, the old code is in history when we implement such a function.

Reply:
I agree. I've also renamed to `AddWidget`.